### DISCUSSION TITLE
Can we use an alias for the module in the defineNuxtConfig?

### DISCUSSION THREAD

**Discussion by baronkoko** [nuxt/nuxt#16861] [Questions] [answered]
- Operating System: `Darwin`
- Node Version:     `v16.14.0`
- Nuxt Version:     `3.0.0-rc.10`
- Nitro Version:    `0.5.3`
- Package Manager:  `npm@8.18.0`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`

**Description**
After updating Nuxt to rc.10, registering a module with an alias `~/`

<img width="476" alt="image" src="https://user-images.githubusercontent.com/27996386/190449932-7f78fccd-6e34-484f-a9c5-ae1e970917f8.png">

causes an error

<img width="657" alt="image" src="https://user-images.githubusercontent.com/27996386/190448917-7f182779-bb77-485c-ad9d-e3a9e0a493ae.png">

Everything works great on stackblitz
https://stackblitz.com/edit/nuxt-starter-fuinuq?file=nuxt.config.ts,module%2Findex.ts

But not locally
https://github.com/baronkoko/nuxt3-module-alias

**Question**
Can we use an alias in the path for the module registration?

**Comment by danielroe** [comment, marked as answer]
You can use an alias; this looks more like an issue with the directory index. Try `~/module/index`.

**Reply by baronkoko** [reply]
It works, thanks 🙏
